src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -428,14 +428,14 @@ void testParentQueryResult() throws Exception {
   void testRejectRequest() {
     UUID id = UUID.randomUUID();
     snapshotBuilderService.rejectRequest(id);
```

Comment:
Sorry to add more work here, but now that this is calling `getById()` you could add a test to verify this
```suggestion
    var response = new SnapshotAccessRequestResponse();
    when(snapshotRequestDao.getById(id)).thenReturn(response);
    assertThat(snapshotBuilderService.rejectRequest(id), is(response));
```

Reply:
good idea!